tasks/configure.yml

```diff
@@ -1,6 +1,20 @@
 ---
-- name: Update config
+- name: Update config v0.9.0
   template: src=influxdb.conf.j2 dest=/etc/opt/influxdb/influxdb.conf
+  when: (influxdb_version.find("rc") != -1 and influxdb_version|version_compare(influxdb_version_thr_0_9_0, '>='))
+        or (influxdb_version.find( "rc" ) == -1 and influxdb_version|version_compare('0.9', '>='))
+  notify:
+    - restart influxdb
+
+- name: Update config elder v0.9.0
```

Comment:
You are not just adding RedHat but also adding back in support for older Influxdb. There are various problems with the older influxdb packages and no path forward for fixes so we purposefully were only supporting 0.9.1+.